### PR TITLE
Fix styling issues from Wagtail 3.0

### PR DESCRIPTION
**Pull request by mmmavis**
This PR addresses the following issues:

### #10192 oversized arrow icon on page explorer
- Using Page Explore to navigate to "Blog" or any directory that has more than 200 pages.
- Make sure arrow icon next to "See all" is sized properly.

### #10194 text in locale dropdown is not visible
- Go to http://test.example.com:8000/cms/snippets/wagtailpages/update/
- Verify text of the locale dropdown is visible.

### CODE REVIEW

File: network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+*/
+
+body#wagtail
+  main#main
+  form
+  ul
+  li
+  .field
+  .c-dropdown.t-inverted.c-dropdown--large
+  a {
```
Comment (mmmavis) [inline network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css:27]: I'm using very specific selector so there's less chance of these styling rules affecting elements that we are unaware of.

File: network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+body#wagtail
+  .sidebar-panel
+  .c-page-explorer__see-more
+  svg.icon.icon-arrow-right {
+  width: 20px;
+  height: 20px;
+  position: relative;
+  top: 5px;
+}
+
```
Comment (danielfmiranda) [inline network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css:14]: Hi @mmmavis!

This is looking great to me so far, after testing, I can confirm that your code changes fix both the issues mentioned in the ticket.

One thing that I did note and wanted to bring up, was that on screen sizes larger than M, it appears that the "see more" box is taller than it needs to be. According to the inspector, it is set to 50px in height (I am assuming to make room for the larger than usual SVG): 


Screenshot 2023-02-24 at 11 46 47 AM


I was wondering what you thought about adding something like this in order to reduce the unnecessary spacing around the text?:

```suggestion

body#wagtail .sidebar-panel .c-page-explorer__see-more {
  height: unset;
}

```

 Here is what it looks like with the suggested CSS above:


Screenshot 2023-02-24 at 11 43 16 AM


I have also tested it on smaller screensizes and can confirm that everything is looking as expected. 

Thanks!

**Comment by mmmavis** [comment]
@danielfmiranda thank you for your awesome suggestion! I've updated the PR with a minor adjustment - I wrapped the styling rules in media query to make sure we are overriding the exact styling rule from Wagtail.

**Review by danielfmiranda — APPROVED** [review]
Thanks for the update Mavis! Approved 👍

**Pull request merged**